content/en/blog/_posts/2021-10-05-nsa-cisa-hardening.md

```diff
@@ -402,11 +402,14 @@ In summary, it is fantastic to see security practitioners sharing this
 level of detailed guidance in public. This guidance further highlights
 Kubernetes going mainstream and how securing Kubernetes clusters and the
 application containers running on Kubernetes continues to need attention and focus of
-practitioners. Only a few weeks after the guidance was published, an open source
-tool [kubescape](https://github.com/armosec/kubescape) to validate cluster
-against this guidance became available.
+practitioners.
```

Comment:
I don't think we should eliminate the lines after practitioners because it would render this factually incorrect. The sentence, IMO, should be retained as is.

```suggestion
practitioners. Only a few weeks after the guidance was published, an open source
tool [kubescape](https://github.com/armosec/kubescape) to validate cluster
against this guidance became available.
```